dataland-frontend/src/frameworks/sfdr/UploadConfig.ts

```diff
@@ -15,7 +15,7 @@ export const sfdrDataModel = [
             name: "dataDate",
             label: "Data Date",
             description: "The date until when the information collected is valid",
-            unit: "",
+
```

Comment:
Wenn ich die toolbox laufen lasse wird in dieser Datei am Anfang ein sinnloser import erzeugt. Ist das erwartet?